### DISCUSSION TITLE
Make next patch release of Istio 1.17 support Kubernetes 1.27

### DISCUSSION THREAD

**Discussion by westwind-sc** [istio/istio#45315] [Q&A]
Hi, Istio 1.17 currently not support Kubernetes 1.27.
And Istio 1.18 will introduce lots of new features and changes, maintainers need more time to release and users need more time to evaluate/upgrade.
To not block upgrading Kubernetes to 1.27, any possibility that Istio officially test and support Kubernetes 1.27 in the next patch release of Istio 1.17?

**Comment by ericvn** [comment]
I did take a look to see if we happened to test with 1.27 in the 1.17 branch today, and I don't see that we do. I believe in the past, it was stated that Istio could attempt to support a new K8s release if it became available before the next minor release. And to claim support we would enable testing of the new k8s release in the pipeline and we would have to run the acceptance test for the .0 (1.17.0 in this case) against a 1.27.x cluster. I can see about adding a job into the pipeline, but the @istio/release-managers-1-17 will need to work with the people running the acceptance tests to see if they can use 1.27.x against 1.17.0. If some changes are needed in a patch release for support a new k8s release, I believe we said that we could call out a specific patch version on as supported as well.

**Reply by ericvn** [reply]
I guess the release managers could check with the person who ran the 1.17.3 acceptance tests what k8s version was used.
